Source/Scene/Scene.js

```diff
@@ -1368,6 +1389,25 @@ define([
                 //>>includeEnd('debug');
                 this._minimumDisableDepthTestDistance = value;
             }
+        },
+
+        /**
+         * Whether or not to use a logarithmic depth buffer. Enabling this option will allow for less frustums in the multi-frustum,
+         * increasing performance. This property relies on {@link Context#fragmentDepth} being supported.
+         */
+        logarithmicDepthBuffer : {
+            get : function() {
+                return this._logDepthBuffer;
+            },
+            set : function(value) {
+                value = this._context.fragmentDepth && value;
+                if (this._logDepthBuffer !== value) {
+                    this._logDepthBuffer = value;
+                    this._logDepthBufferDirty = true;
+
+                    OPAQUE_FRUSTUM_NEAR_OFFSET = this._logDepthBuffer ? 0.9 : 0.9999;
```

Comment:
A bit sketchy since (1) this assigns to a constant that isn't really a constant and (2) this setter function needs to be called for everything to be initialized, which leads to a less cohesive constructor function.

Perhaps replace `OPAQUE_FRUSTUM_NEAR_OFFSET` with a getter function.